source/features/index.tsx

```diff
@@ -284,6 +290,7 @@ const features = {
 	add,
 	addCssFeature,
 	error: logError,
```

Comment:
I think what @fregante meant was to remove the `error` here and update where ever we use `features.error` to be `features.log.error`.

Reply:
Thank you for the guidance. Done.

Reply:
Your welcome.
You forgot to remove the export.

Reply:
Which export are you referring to?

In `features/index.tsx` there's only the initial one:

https://github.com/sindresorhus/refined-github/blob/3262b37f77f12f404e57f5434c2324f1ed41ffad/source/features/index.tsx#L292

which cannot be removed, and XO suggests to keep it default, i.e. not make it non-default.

Reply:
My bad everything is good, I misread the diff.

🎉